src/agent/Cargo.toml

```diff
@@ -35,7 +35,7 @@ tokio = { version = "1.14.0", features = ["full"] }
 tokio-vsock = "0.3.1"
 
 netlink-sys = { version = "0.7.0", features = ["tokio_socket",]}
-rtnetlink = "0.11.0"
+rtnetlink = "0.8.0"
```

Comment:
Do we have to revert this?

Reply:
The commit that I reverted is this: https://github.com/kata-containers/kata-containers/pull/4770/commits/845c1c03cfbe8435ac383964ef243da252726351

It upgraded rtnetlink to 0.11.0 and used the new API for adding neighbours. Hence the revert includes going back to previous version. We can keep the newer version while still using the older API or upgrade to newer version of rtnetlink when use of the new API is added back again. I am ok with either. Let me know.

Reply:
If the newer version works fine with the older API, I'd go for that.

Reply:
@fidencio Doesnt seem to work.
```
error[E0599]: no method named `set_name_filter` found for struct `LinkGetRequest` in the current scope
   --> src/netlink.rs:167:47
    |
167 |             LinkFilter::Name(name) => request.set_name_filter(name.to_owned()),
    |                                               ^^^^^^^^^^^^^^^ method not found in `LinkGetRequest
```